src/mumble/MainWindow.cpp

```diff
@@ -1969,6 +1973,20 @@ void MainWindow::openTextMessageDialog(ClientUser *p) {
 	delete texm;
 }
 
+void MainWindow::on_qaUserLocalNickname_triggered() {
+	ClientUser *p = getContextMenuUser();
+
+	if (!p)
+		return;
+
+	openUserLocalNicknameDialog(p);
+}
+
+void MainWindow::openUserLocalNicknameDialog(ClientUser *p) {
+	unsigned int session = p->uiSession;
```

Comment:
```suggestion
	const auto session = p->uiSession;
```

Reply:
I'm against using `uato` in this case. I much prefer knowing what the type of the variable is just by looking at the code (without the need of an IDE)